examples/draft-pad/DEPLOYMENT.md

```diff
@@ -0,0 +1,187 @@
+# Deploying Draft-Pad to Vercel with GenSX Workflows
+
+This guide explains how to deploy the Draft-Pad application to Vercel with automatic GenSX workflow deployment.
+
+## Prerequisites
+
+1. A Vercel account
+2. A GenSX account with API access
+3. The GenSX CLI installed (it will be installed automatically via npx)
+
+## Setup Steps
+
+### 1. Configure GenSX Authentication
+
+First, you need to authenticate with GenSX locally to create your project:
+
+```bash
+npx gensx@latest login
+```
+
+### 2. Create Your GenSX Project
+
+Create the project in GenSX Cloud:
+
+```bash
+npx gensx@latest project create draft-pad
+```
+
+### 3. Set Environment Variables in Vercel
+
+In your Vercel project settings, add the following environment variables:
+
+- `GENSX_API_KEY`: Your GenSX API key (get this from the GenSX dashboard)
+- `GENSX_TOKEN`: Your GenSX auth token (get this from `~/.gensx/credentials.json` after login)
+- `NODE_ENV`: Set to `production`
+
+Optional:
+
+- `GENSX_BASE_URL`: Only if using a custom GenSX instance (defaults to https://api.gensx.com)
+
+### 4. Update Your Deployment Configuration
+
+The `package.json` has been configured to automatically deploy workflows during the build process:
+
+```json
+{
+  "scripts": {
+    "build": "npm run deploy-workflow && next build",
+    "deploy-workflow": "npx gensx@latest deploy ./gensx/workflows.ts --project draft-pad --env production --yes"
+  }
+}
+```
+
+### 5. Configure GenSX CLI for CI/CD
+
+Since the GenSX CLI needs authentication in the CI environment, create a `.gensx/credentials.json` file structure:
+
+```bash
+mkdir -p .gensx
+```
+
+Then create a build script that sets up credentials:
+
+The `scripts/setup-gensx-ci.sh` script will automatically set up GenSX credentials from environment variables during the build process.
```

Comment:
What is this?

Reply:
I removed this file. Has a lot of errant crap in there.